test/TestAssets/TestProjects/TestAppSimple/TestAppSimple.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
+
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
```

Comment:
I like these changes, thank you ;)

Reply:
As mentioned above, I didn't change this. This is what is currently there: https://github.com/dotnet/sdk/blob/main/src/Assets/TestProjects/TestAppSimple/TestAppSimple.csproj